apprentice-react-o-shop/.gitignore

```diff
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```

Comment:
let's clean this file up to only ignore things that we're using. The less lines of code to accomplish what you want the cleaner. I think you probably only need .node_modules